tests/auth.test/t09.expected

```diff
@@ -114,6 +115,7 @@
 (candidate='LZ4')
 (candidate='MATCH')
 (candidate='NATURAL')
+(candidate='NEXTSEQUENCE')
```

Comment:
Drop this and only keep autoinc?

Reply:
It ends up conflicting with sqlite's autoincrement- I will document this function though.